src/main/java/com/nirmata/workflow/details/internalmodels/StartedTask.java

```diff
@@ -39,6 +41,16 @@ public LocalDateTime getStartDateUtc()
     {
         return startDateUtc;
     }
+    
+    public int getProgress()
+    {
+        return progress;
+    }
+    
+    public void setProgress(int progress)
```

Comment:
Please follow the model of the rest of the library. Create a new instance with the new progress value. It can be done either by having setProgress return a new StartedTask or by constructing a new StartedTask via constructor.